CHANGELOG.md

```diff
@@ -24,7 +24,8 @@ All notable changes to this project will be documented in this file.
 - The connector public key is now also persisted.
 - Set URI-converter and column-length for location in AppStore-entity. Tested AppStore-App deployment with Portainer 2.14.0
 - Query parameters are ignored if a query parameter with the same key is already defined in the target url
-
+- Hyphen in docker-compose led to unexpected character exception
+-
```

Comment:
Could you remove this extra line? It is empty otherwise:

![image](https://user-images.githubusercontent.com/75306992/179539515-a81663ac-1f31-44ec-8c06-2781d281d93c.png)